internal/models/kubeconfig_setting.go

```diff
@@ -16,6 +16,7 @@ type KubeconfigSetting struct {
 	AccountId                   uuid.UUID `bun:"account_id,type:uuid,notnull"`
 	Scope                       string    `bun:"scope,notnull"`
 	ValiditySeconds             int64     `bun:"validity_seconds,notnull,default:0"`
+	SaValiditySeconds           int64     `bun:"sa_validity_seconds,notnull,default:0"`
```

Comment:
we decided the default will be eight hours right? so it should be the corresponding value in seconds.